pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about-assemblies.adoc

```diff
@@ -5,13 +5,21 @@
 
 
 [id="con_about-assemblies_{context}"]
-= About Assemblies
+= Overview of assemblies
+:_module-type: CONCEPT
 
 [role="_abstract"]
-In {ProductName} {ProdctNumber}, each assembly encompasses a single user story. An assembly can include concept, procedure, or reference modules but all modules in an assembly must relate to that specific user story. Previously, many Red Hat documentation teams used assemblies as the structure of user guides and other "books." In {ProductName} {ProductNumber}, there are no books, only user stories, with one assembly per user story. Focusing on user stories is part of the Flexible Customer Content (FCC) effort at Red Hat. One of the goals of this approach is to make it easier for users to search for each specific action they want to perform.
+In {ProductName} {ProdctNumber}, each assembly encompasses a single user story. Previously, an assembly could include multiple user stories, with only one assembly being used for a complete user guide. However, in {ProductName} {ProductNumber}, each user story requires a separate guide.
```

Comment:
@RichardHoch Sometimes a user story (assembly) requires other user stories. The Mod docs reference guide shows an example of this. So I don't know if this is completely accurate.